definitions/npm/react-i18next_v1.10.x/flow_v0.36.x-/react-i18next_v1.10.x.js

```diff
@@ -0,0 +1,35 @@
+import type { TFunction } from 'i18next'
```

Comment:
if tests are in place, I think we would be ready to merge... I saw a lot of imports in other libdefs now, so maybe I am just wrong here :-)

If you still need any help, just ping me